molo/usermetadata/views.py

```diff
@@ -0,0 +1,29 @@
+from django.views.generic import TemplateView
+from django.shortcuts import get_object_or_404
+from django.http import HttpResponseRedirect
+
+from molo.usermetadata.models import PersonaPage
+
+
+class PersonaView(TemplateView):
+    template_name = 'persona.html'
+
+    def get_context_data(self, *args, **kwargs):
+        persona_pages = PersonaPage.objects.live().all()
+        context = super(PersonaView, self).get_context_data(*args, **kwargs)
+        context.update({
+            'persona_pages': persona_pages,
```

Comment:
@codiebeulaine please also add tests for the translated persona pages as well :)